example/variable_raycast_collisions/data/player.script

```diff
@@ -0,0 +1,51 @@
+local platypus = require"platypus.platypus"
```

Comment:
Indentation is using spaces not tabs. Please fix this to be consistent with the rest of the code.

Reply:
done